IntegrationTests/ControllerTests/AccountControllerTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿using AdminDashboard.Main;
```

Comment:
Below are all of the integration tests (for all the happy paths) and unit tests for the complex logic